README.md

```diff
@@ -236,8 +236,14 @@ A few advantages of this solution:
 ## Develop
 
 1. Write code and tests.
-1. Make sure all tests pass `npm test`
-1. Make sure code is well formatted and secure `npm run lint`
+2. Make sure all tests pass `npm test`
+
+If in-case some tests still fails randomly during local development, the reason to it might be when you develop something in the parser and even though you did not make any changes affects tests, and the tests are failing then you can run:
```

Comment:
yes we can add that also then we need to add sub-sections for Mac or Linux users as well then right?

Reply:
Not unless you have something specific you want to add there 🙂 To me a section only makes sense if there is something to highlight, i.e. in this case windows environment might act weird in some cases.

Reply:
agreed, would do the changes in the PR

Reply:
@jonaslagoni  Totally, if it's just for `Windows` OS, let's def add a `### Windows` title.

(I don't think you need to write the word 'environment' too, it's pretty clear to just say **Windows**.)

Reply:
We need to change this sentence I think: `the reason to it might be when you develop something in the parser and even though you did not make any changes affects tests`

To me it is not clear what exactly the reason is. Maybe something along the lines of:
```suggestion
For Windows environments, some tests might still fail randomly during local development even when you made no changes to the tests. The reason for this from file endings are different than expected and this comes from Git defaulting to an unexpected file ending. If you encounter this issue you can run the following commands to set Git to use the expected one:
```

What do you think @Ruchip16 ?

Reply:
yess absolutely the later one gives more clarity than its prior, I would do the changes